pkg/controller/controller.go

```diff
@@ -101,6 +101,11 @@ const (
 	nodePublishSecretNameKey      = "csiNodePublishSecretName"
 	nodePublishSecretNamespaceKey = "csiNodePublishSecretNamespace"
 
+	// PVC's name and namespace passed on create requests, useful for the plugin to query for extra data
+	// prior to performing operations
+	pvcNameKey      = "kubernetes.io/created-for/pvc/name"
```

Comment:
can we also add pv name as one of these?

Reply:
We thought about this but, the PV name is generated in the provisioner and passed in the request as the Name field already and always, do you see any need for adding it into the parameters also?

Reply:
would prefer it is added to the parameters as well. Using it as the name is just an implementation choice in the provisioner not an explicit contract so it could technically change.